test/unit/iam-token-manager.test.js

```diff
@@ -62,14 +63,11 @@ describe('iam_token_manager_v1', function() {
     const requestMock = jest.spyOn(instance, 'requestToken');
 
     const currentTokenInfo = {
-      access_token: '1234',
-      refresh_token: '5678',
-      token_type: 'Bearer',
-      expires_in: 3600,
-      expiration: Math.floor(Date.now() / 1000),
+      access_token: CURRENT_ACCESS_TOKEN,
```

Comment:
Since the `iam-token-manager` was changed to simply treat the access token as a JWT, none of the other fields in the IAM response are used, so we can eliminate them.